packages/bitcoin/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@chainflip/bitcoin",
   "version": "1.2.4",
-  "type": "module",
```

Comment:
if we set `"type": "module"`, the package is treated as esm module. esm modules are required to have a file extension for relative imports (eg `./utils.js` instead of `./utils`). so we have two options:
- remove the `"type": "module"`
- change all the imports

i chose option 2 because this is consistent to most of our other repos, but its not a strong opinion